pkgs/development/python-modules/channels_redis/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, redis, msgpack, channels, aioredis, async_generator, pytest
+}:
+buildPythonPackage rec {
+
+  pname = "channels_redis";
+  version = "2.4.0";
+
```

Comment:
Please use a `disabled` parameter so this package can be disabled for the following python versions https://github.com/django/channels_redis/blob/master/setup.py#L32